### DISCUSSION TITLE
230803 클라이언트 api 요청 체크

### DISCUSSION THREAD

**Discussion by 0gunkim** [mi-gan-zi/FE-MiGanZi#126] [Check]
## Board
|Route|Method |API|작동 여부|이슈|이슈내용|링크
|:---:|:---:|:---|:---:|:---:|:---:|:---:|
|`/create`| `POST`|`/user/board/post/write`|o|x|x|x|
|`/detail/:id`|`GET`|`/user/board/{id}`|x|x|x|x|
|`/`|`GET`|`/user/board/posts`|x|x|x|x|
|`/`|`GET`|`/user/board/popular-post`|x|x|x|x|

### Search
|Route|Method |API|작동 여부|이슈|이슈내용|링크
|:---:|:---:|:---|:---:|:---:|:---:|:---:|
|`/search`| `GET`|`/user/board/find-near-post/{lat}/{lng}/{tags}`|x|x|x|x|

### Comment
|Route|Method |API|작동 여부|이슈|이슈내용|링크
|:---:|:---:|:---|:---:|:---:|:---:|:---:|
|`/detail/:id`|`POST`|`/user/board/comment/write`|x|x|x|x|

## User
|Route|Method |API|작동 여부|이슈|이슈내용|링크
|:---:|:---:|:---|:---:|:---:|:---:|:---:|
|`/login`|`POST`|`/user/login`|o|x|x|x|
|`/logout`|`POST`|`/user/board/post/write`|o|x|x|x|

**Comment by new-crystal** [comment]
<img width="1372" alt="스크린샷 2023-08-03 오후 8 14 50" src="https://github.com/mi-gan-zi/FE-MiGanZi/assets/109053875/d49b91cc-e7ec-4ea3-a752-c64dec6c2cf3">
<img width="1384" alt="스크린샷 2023-08-03 오후 8 11 49" src="https://github.com/mi-gan-zi/FE-MiGanZi/assets/109053875/cfae5dc4-c277-4632-8462-c76a181a9460">
<img width="1273" alt="스크린샷 2023-08-03 오후 7 49 05" src="https://github.com/mi-gan-zi/FE-MiGanZi/assets/109053875/9d943050-c9b0-4b0c-81d4-d11f1fd42e7f">
<img width="1050" alt="스크린샷 2023-08-03 오후 7 43 28" src="https://github.com/mi-gan-zi/FE-MiGanZi/assets/109053875/2366e790-ab6c-48b6-8213-21d5c8158e05">
<img width="1323" alt="스크린샷 2023-08-03 오후 7 42 40" src="https://github.com/mi-gan-zi/FE-MiGanZi/assets/109053875/4ec86424-b3ae-4377-93e3-dd18aa10906c">
<img width="959" alt="스크린샷 2023-08-03 오후 7 43 00" src="https://github.com/mi-gan-zi/FE-MiGanZi/assets/109053875/8bca2c35-dd26-4648-98c5-3a31ad8e4f04">

**Comment by evga7** [comment]
백앤드 자체 테스트 결과 입니다.
https://github.com/mi-gan-zi/BE-MiGanZi/discussions/64
